openfisca_us/parameters/hhs/tanf/max_amounts.yaml

```diff
@@ -0,0 +1,47 @@
+description: Payment levels for TANF benefit calculations.
+IL:  
+  1: 
+    2021-10-01: 322
+  2:
+    2021-10-01: 435
+  3: 
+    2021-10-01: 549
+  4: 
+    2021-10-01: 662
+  5: 
+    2021-10-01: 776
+  6: 
+    2021-10-01: 890
+  7: 
+    2021-10-01: 1003
+  8: 
+    2021-10-01: 1116
+  9:
+    2021-10-01: 1230
+  10:
+    2021-10-01: 1343
+  11:
+    2021-10-01: 1457
+  12:
+    2021-10-01: 1571
+  13:
+    2021-10-01: 1684
+  14:
+    2021-10-01: 1797
+  15:
+    2021-10-01: 1911
+  16:
+    2021-10-01: 2024
+  17:
+    2021-10-01: 2138
+  18:
+    2021-10-01: 2252
+metadata:
+  unit: currency-USD
+  reference: 
+  - title: "Illinois TANF Eligibility and Benefit Amount: WAG 25-03-05: Table of Monthly Allowances"
+    href: https://www.dhs.state.il.us/page.aspx?item=12673
```

Comment:
Could you add a comment showing which page indicates that these values took effect on 2021-10-01?